roles/sap_hana_install/defaults/main.yml

```diff
@@ -70,6 +70,9 @@ sap_hana_install_check_installation: no
 # no: use the command `hdblcm --action=check_installation`
 sap_hana_install_use_hdbcheck: yes
 
+# If the following variable is set to yes the idempotency check, if HANA is already installed will be skipped
```

Comment:
meaning unclear, please modify.
Do you mean the following?
--->
If the following variable is set to yes, the HANA installation check will be skipped
<---

Reply:
yes thanks